bridge/slack/helpers.go

```diff
@@ -64,6 +64,7 @@ func (b *Bslack) getChannelBy(lookupKey string, lookupMap map[string]*slack.Chan
 const minimumRefreshInterval = 10 * time.Second
 
 func (b *Bslack) populateUsers() {
+	time.Sleep(time.Second)
```

Comment:
What is this doing here? Same question for the `populateChannels()` method below.

Reply:
Ah shoot. This shouldn't be here. Thanks! (It's my workaround for the users lookup otherwise becoming empty on large slack teams, for which I haven't had time to track down root cause.)